package.json

```diff
@@ -17,6 +17,7 @@
     "build:compile": "babel --source-maps -d lib src",
     "build:bundle": "NODE_ENV=production webpack -p lib/vector/index.js vector/bundle.js",
     "build": "npm run build:css && npm run build:compile && npm run build:bundle",
+    "package": "npm run build && mkdir packages && bestzip packages/vector-`git describe --dirty || echo unknown`.zip vector",
```

Comment:
you want `mkdir -p`.

Also, I'm led to wonder why we bother with `bestzip`, given we're assuming other linuxy things like `mkdir` and `git` and `&&`, but whatever.